app/templates/partials/available_in_language.html

```diff
@@ -0,0 +1,13 @@
+{% macro available_in_language(inset, warn) %}
```

Comment:
With boolean traps, in python I'd force keyword arguments by swallowing positional arguments:

```python
def available_in_language(*, inset, warn):
    ...
````

Which would throw an error unless you explicitly specify keywords:
```
available_in_language(True, True)
  TypeError: available_in_language() takes 0 positional arguments but 2 were given
```

This doesn't seem to be available for jinja macros however :(